app/dashboard/utils.py

```diff
@@ -468,6 +468,7 @@ def get_context(ref_object=None, github_username='', user=None, confirm_time_min
         'handle': getattr(user, 'username', ''),
         'title': title,
         'gas_advisories': gas_advisories(),
+        'gitcoin_tip_address': gitcoin_tip_address
```

Comment:
Please end with a trailing comma according to the [PEP-8 guidance](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)